config/test.toml

```diff
@@ -1,4 +1,4 @@
-owner = "bnjbvr"
+owner = "davidpdrsn"
```

Comment:
We need a test repo in the organization... That requires a different kind of GitHub token though. A job for another day.